src/util/text_parser.ts

```diff
@@ -101,3 +106,20 @@ export const extractTasks = (body: string) => {
   });
   return tasks;
 };
+
+export const extractIntegrationDocumentationLinks = (
+  body: string
+): IntegrationDocumentationLink[] => {
+  const re = /https:\/\/(www.|rc.|next.|)home-assistant.io\/integrations\/(\w+)/g;
```

Comment:
Should the `.` be escaped?
```suggestion
  const re = /https:\/\/(www\.|rc\.|next\.|)home-assistant.io\/integrations\/(\w+)/g;
```

Does \w+ also match underscore `_` ?

Reply:
If you don't escape `.` it matches any character except linebreaks

And yes, \w+ includes `_`, I can add a test for that